extensions/schema-compare/src/schemaCompareMainWindow.ts

```diff
@@ -1151,8 +1151,8 @@ export class SchemaCompareMainWindow {
 				connectionDetails: undefined,
 				projectFilePath: endpoint.projectFilePath,
 				targetScripts: [],
-				dataSchemaProvider: '',
-				folderStructure: mssql.ExtractTarget.schemaObjectType			// TODO: Pick this automatically from the scmp file, after issue #20332 is resolved (check dsp as well)
+				dataSchemaProvider: endpoint.dataSchemaProvider,
+				extractTarget: endpoint.extractTarget			// TODO: Pick this automatically from the scmp file, after issue #20332 is resolved (check dsp as well)
```

Comment:
Is this PR resolving this `TODO`/can the comment be removed?

Reply:
My bad, this PR addresses this itself and it completely slipped off ! Fixed it.